examples/tcp-client/tcp_client.out

```diff
@@ -1,7 +1,6 @@
 # To run the client, navigate to the directory that contains the
```

Comment:
As a prerequisite, should we say to run the tcp_listener sample first?

Reply:
+1